### ISSUE TITLE
add a link to an 'apps list' in sign in page

### ISSUE THREAD

**Issue by pravi** [diaspora/diaspora#6858]
Currently Sign In page have "Forgot Password" and "Sign Up". We should add a link to apps page. The apps page should list mobile apps for diaspora and link to Play Store/Fdroid. When chat is enabled it should show XMPP apps and mention any of these could be used for real time chat.

**Comment by denschub** [comment]
I disagree. Linking apps in the default template would imply that we endorse or support those applications, which, in fact, we don't. I do not think we should promote applications that scrape the html contents for csfr_tokens or but diaspora into a webview with added hacks since those are prone to fail at any time... Same story goes for the XMPP chat, which is not even stabilized yet and anything may change in the future.

There is a page in the wiki mentioning various third-party development efforts and maybe we should link that page somwhere inside diaspora\*, maybe in the help section. But linking them in a very prominent place feels wrong to me.

$ 0.02

**Comment by pravi** [comment]
How about adding it after we have API (apps using that) and stable chat?

**Comment by denschub** [comment]
Sure, but that's some time far in the future. :)

**Comment by pravi** [comment]
The idea just occured to me and I thought of noting it somewhere before I forget :)

**Comment by Flaburgan** [comment]
You've a good point about XMPP app though. We have a "chat" section in the help of diaspora*, I encourage you to open a PR explaining that the chat uses XMPP and therefore any XMPP native app can be used on mobile.
